bundles/org.openhab.binding.adorne/src/main/feature/feature.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<features name="org.openhab.binding.adorne-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
-    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
+<features name="org.openhab.binding.adorne-${project.version}"
```

Comment:
This is due to an old version of the spotless configuration. Please have one line per tag, regardless of the line-length. Otherwise the feature-aggregation will fail.

Reply:
Done

Reply:
Still on two lines

```suggestion
<features name="org.openhab.binding.adorne-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Reply:
sorry, didn't push yet - unresolving comment.

Reply:
Pushed